tests/Validation/ValidationValidatorTest.php

```diff
@@ -4392,6 +4392,18 @@ public function testValidateSometimesImplicitEachWithAsterisksBeforeAndAfter()
         });
         $this->assertTrue($v->passes());
 
+        return;
```

Comment:
Is this `return` supposed to be here? Seems like it is exiting the test prematurely.